translate/src/modules/search/components/SearchBox.tsx

```diff
@@ -38,7 +39,12 @@ type InternalProps = Props & {
   dispatch: AppDispatch;
 };
 
-export type FilterType = 'authors' | 'extras' | 'statuses' | 'tags';
+export type FilterType =
+  | 'authors'
+  | 'extras'
+  | 'statuses'
+  | 'search_identifiers'
```

Comment:
I find it confusing that Filters and Search options are two separate things in the UI, but in the code we mix them.

How difficult would it be to move search options out of `FilterType` and `FilterState`, and then also stop calling `applyFilters` and `toggleFilter` from the `SearchPanel`?